raster-foundry/raster-foundry
# Update platform email settings and ingest email notification

## Overview

This PR adds port, encryption method, and export notification fields to platform settings both backend and frontend to support more flexible email sending service and the export notification job in batch (which will be implemented in #3537). It also updates the existing ingest notification methods to accomodate new fields.

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Notes

This PR is opened since these new email fields affect #3552. It is also a part of #3537 

## Testing Instructions

 * Run migration
 * Make yourself platform admin or super user and go to platform setting page
 * Update your settings and make sure it works as you intend it to
 * Refer to the testing instructions in https://github.com/raster-foundry/raster-foundry/pull/3525 for sending notification emails about ingest status and make sure it still works

Closes #3577
